chef/site-cookbooks/wca/recipes/cronjobs.rb

```diff
@@ -1,24 +1,29 @@
 username, repo_root = WcaHelper.get_username_and_repo_root(self)
+secrets = WcaHelper.get_secrets(self)
 
 admin_email = "admin@worldcubeassociation.org"
 path = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
 
-db_dump_folder = "#{repo_root}/secrets/wca_db"
-dump_command = "#{repo_root}/scripts/db.sh dump #{db_dump_folder}"
+secrets_folder = "#{repo_root}/secrets"
+db_dump_folder = "#{secrets_folder}/wca_db"
+dump_db_command = "#{repo_root}/scripts/db.sh dump #{db_dump_folder}"
+dump_gh_command = "github-backup --incremental --all -t #{secrets['GITHUB_BACKUP_ACCESS_TOKEN']} --organization thewca -o #{secrets_folder}/github-thewca"
```

Comment:
Is it possible we might have any forks in the future? I'd suggest passing `--fork` just in case.

(Same for `--private`, except that `thewca` doesn't have private repos, I assume.)

I also notice you added `--incremental`, which I forgot to include in my email. Good on you. :-)